templates/index.html

```diff
@@ -3,18 +3,39 @@
 {% block title%}Home Page{% endblock%}
 {% block content %}
 
-<div class="container">
+
+<div class="container" id="poop">
```

Comment:
... what?

Reply:
@IWantWaffleFries1999 this would be the poop you mentioned earlier lol

Which, is also totally common. I had a contractor at work add a debug line that said "got here, biyatch!!!" and he forgot to remove i before attempting to commit.